### PR TITLE
Remove "virtual" wording for bare metal resource requirements

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: https://deploy-preview-38651--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#minimum-resource-requirements_installing-bare-metal

Non-bare metal preview (for comparison): https://deploy-preview-38651--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#minimum-resource-requirements_installing-vmc-user-infra

### CODE REVIEW

**Comment by codyhoag** [comment]
@pamoedom can you verify? Thanks!

File: modules/installation-requirements-user-infra.adoc [hunk 1]
```diff
@@ -194,11 +200,14 @@ endif::openshift-origin[]
 ifdef::ibm-z[]
 1. One physical core (IFL) provides two logical cores (threads) when SMT-2 is enabled. The hypervisor can provide two or more vCPUs.
 endif::ibm-z[]
-ifndef::ibm-z[]
+ifdef::bare-metal[]
+1. One CPU is equivalent to one physical core when simultaneous multithreading (SMT), or hyperthreading, is not enabled. When enabled, use the following formula to calculate the corresponding ratio: (threads per core × cores) × sockets = CPUs.
+endif::bare-metal[]
+ifndef::ibm-z,bare-metal[]
```
Comment (pamoedom) [inline modules/installation-requirements-user-infra.adoc:206]: Hi @codyhoag, in principle LGTM but I think you should also add here the previous notes about IOPS and RHEL, right?

~~~
2. {product-title} and Kubernetes are sensitive to disk performance, and faster storage is recommended, particularly for etcd on the control plane nodes which require a 10 ms p99 fsync duration. Note that on many cloud platforms, storage size and IOPS scale together, so you might need to over-allocate storage volume to obtain sufficient performance.
3. As with all user-provisioned installations, if you choose to use {op-system-base} compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks. Use of {op-system-base} 7 compute machines is deprecated and planned for removal in a future release of {product-title} 4.
~~~
Regards.

Comment (codyhoag) [reply modules/installation-requirements-user-infra.adoc:206]: Yeah, good call. I added those back for bare metal :+1: . Thanks!

**Review by kalexand-rh — COMMENTED** [review]
LGTM!

**Pull request merged**